src/TypeScript/EslintBridgeClient/EslintBridgeClient.cs

```diff
@@ -114,13 +114,19 @@ public async Task Close()
         {
             try
             {
-                await MakeCall("close", null, CancellationToken.None);
+                if (eslintBridgeProcess.IsRunning)
```

Comment:
This is not wrapped in a lock, but I can't think of a scenario in which thread A will be in the process of of calling "close" and thread B will already get to "eslintBridgeProcess.Stop()" -- there is only one instance of this class per analyzer, and the analyzer calls this method only when solution closes / vs shuts down. Both operations are very slow so I don't think a lock is needed. Let me know if you can think of a use case.